### PR TITLE
[bitnami/external-dns] PDB review

### PR DESCRIPTION
**Pull request by fmulero**
### Description of the change

Little fixes in PodDisruptionBudget configuration to keep it aligned with current templates.

### Benefits

Keep our charts up to date according to our templates

### Possible drawbacks

Old `podDisruptionBudget` value was deprecated.

### Adition information

* [Pod disruption budgets](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets)
* [Specifying a Disruption Budget for your Application](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/external-dns/values.yaml [hunk 1]
```diff
@@ -1205,11 +1205,17 @@ extraVolumes: []
...
 ##
 extraVolumeMounts: []
-## @param podDisruptionBudget Configure PodDisruptionBudget
-## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
-##
-
-podDisruptionBudget: {}
+## Pod Disruption Budget configuration
+## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb
+## DEPRECATED: podDisruptionBudget will be removed in a future release - Please use pdb instead
```
Comment (juan131) [inline bitnami/external-dns/values.yaml:1210]: You shouldn't remove if you're deprecating it: 
```suggestion
## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb
## DEPRECATED: podDisruptionBudget will be removed in a future release - Please use pdb instead
podDisruptionBudget: {}
```

**Review by juan131 — CHANGES_REQUESTED** [review]

File: bitnami/external-dns/values.yaml (hunk 1 above)
Comment (fmulero) [reply bitnami/external-dns/values.yaml]: Suggestion applied. I also set `pdb.create=true` by default

**Review by juan131 — APPROVED** [review]
LGTM

**Comment by rouke-broersma** [comment]
@fmulero @juan131 this new default pdb configuration has two major issues. See #26199

**Pull request merged**